### PR TITLE
Instructions for local doc build

### PR DESCRIPTION
**Pull request by bobturneruk**
Resolves https://github.com/ScottishCovidResponse/SCRCIssueTracking/issues/405

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,9 +183,25 @@ optional arguments:
   --infected INFECTED   Number of infected people in each region/age group
                         (default: 100)
 ```
- 
+
 Descriptions of the data files used can be found in the [data dictionary](sample_input_files/README.md).
 
+## Building the documentation
+
+To build a local version of the readthedocs site e.g. when working on the documentation.
```
Comment (alex-konovalov) [inline README.md:191]: Perhaps one could rephrase this to avoid terminating an incomplete sentence with a full stop.

**Comment by alex-konovalov** [comment]
@bobturneruk now this PR has conflicts which should be resolved.

**Comment by bobturneruk** [comment]
Superseded by https://github.com/ScottishCovidResponse/simple_network_sim/pull/74
